src/Servers/Kestrel/test/InMemory.FunctionalTests/ChunkedResponseTests.cs

```diff
@@ -517,6 +526,8 @@ public async Task ChunksWithGetMemoryLargeWriteBeforeFirstFlush()
         [Fact]
         public async Task ChunksWithGetMemoryWithInitialFlushWorks()
         {
+            var length = new IntAsRef();
```

Comment:
@benaadams, why did you need this? Isn't local capturing enough?

Reply:
Didn't trust it :) Was just being explicit that that was the intent